data/calendar.go

```diff
@@ -45,6 +45,7 @@ const (
 	EventTagActions
 	EventTagCancelled
 	EventTagCancelable
+	EventTagSection
```

Comment:
When this is merged, you should also be sure to update the API docs with the new tag.

Reply:
Will do!